enhancements/operator-conditions-lib.md

```diff
@@ -98,49 +157,17 @@ In this case, the specific API will return an error. The returned error can be w
-- `*apiv1alpha1.Status` refers to the CR definition provided by OLM and
-- [`metav1.Condition`][metav1-Condition] refers to specific Condition type present in `status.conditions` array.
+### Advantage of using this format:
+1. Users need not import `operator-framework/api` to modify the conditions in the CR.
+2. Users can create a new interface which implements `ConditionManager` for custom conditions which are not supported by OLM.
 
-```go
-// GetNamespacedName - returns the NamespacedName of the CR.
-// Will return error when the namespace or name of the associated
-// Status CR cannot be found.
-GetNamespacedName() (types.NamespacedName, error)
-
-// SetConditionStatus - adds the specific condition to Status CR or updates the condition.Status
-// if the particular condition is already present.
-// Errors will be retured as specified in the `Scenarios` section.
-SetConditionStatus(status *apiv1alpha1.Status, condition metav1.Condition) error
-
-// RemoveConditionStatus - remove the specific condition present in Status CR.
-// Will return an error if the specified condition was not found in Status.
-RemoveConditionStatus(status *apiv1alpha1.Status, condition metav1.Condition) error
-
-// FindConditionStatus - finds the ConditionType in `status.conditions` array of Status CR.
-// Return nil if not present.
-FindConditionStatus(status *apiv1alpha1.Status, conditionType string) *metav1.Condition
-
-// IsConditionStatusTrue - returns true when the conditionType is present in `status.conditions`
-// and is set to `metav1.ConditionTrue`
-IsConditionStatusTrue(status *apiv1alpha1.Status, conditionType string) bool
-
-// IsStatusConditionPresentAndEqual - returns true if conditionType is present in
-// `status.conditions` and status is equal to as provided.
-IsConditionStatusPresentAndEqual(status *apiv1alpha1.Status, conditionType string, conditionStatus metav1.ConditionStatus) bool
-```
+### Disadvantages of using this format:
+1. In order to create/add a new condition, users will have to be directed to utilize the helpers provided in [apimachinery][apimachinery_condition]. 
 
 ### Location of the library:
 
 This library will live in [`operator-lib`][operator_lib] repository and the definitions for Status CRD will be present in [`operator-framework/api`][api_repo]. OLM will utilize the `Condition` type defined in [apimachinery][apimachinery_condition]. The existing [`status`][operator_lib_status] package present in operator-lib will be removed in favor of the available helpers to work with conditions provided in [apimachinery][apimachinery_repo].
 
-**NOTE:**
-Operators cannot CREATE/DELETE the `Status` CR. OLM will scope the operators' RBAC to only GET, UPDATE, PATCH the Status CR associated with their operator.
-
-### Risks and Mitigations
-
-- In order to restrict modifications on entire Status CR, OLM will restrict operators' changes to only [`status`][status-sub-resource] sub-resource, though the entire CRD can be read.
-
```

Comment:
These are important details that (I think) are unchanged based on our recent discussions. Any reason these are removed?